akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -43,7 +43,15 @@ object ExceptionHandler {
         ctx.complete((status, info.format(settings.verboseErrorMessages)))
       }
       case NonFatal(e) ⇒ ctx ⇒ {
-        ctx.log.error("Error during processing of request: '{}'. Completing with {} response.", e.getMessage, InternalServerError)
+        val message = Option(e.getMessage).getOrElse(s"${e.getClass.getName} (No error message supplied)")
+        val template =
+          s"""Error during processing of request: '{}'.
+             |Completing with {} response.
+             |Stack traces aren't printed by default.
+             |To customize this behavior, consider implementing a custom ExceptionHandler
+             |(http://doc.akka.io/docs/akka-http/current/scala/http/routing-dsl/exception-handling.html)"""
```

Comment:
I would not add the link to the docs in here. Structure may change, or hosted somewhere else, and this is the last thing you remember to modify when this happens.
Furthermore, the Java `ExceptionHandler` is just a wrapper on top of the Scala one, so even for Java API users it will point to the Scala documentation page.